packages/react-charting/docs/TestPlans/LineChart/ComponentTests.md

```diff
@@ -0,0 +1,67 @@
+**Line Chart – Component test plan**
+
+**Sub-components: Line, x-axis, y-axis, Event, Time Range, Callout, Drop down and Legend**
+
+1. **Line: Line data, Line color (multi colors), Line label (show/hide)**
```

Comment:
Add tests for time range, event annotations, gaps in line and custom date axis locale.

Reply:
Sure i will add remaining tests

Reply:
Added tests for Time range and gaps